docs/page/gallery.md

```diff
@@ -17,10 +17,6 @@ Gallery
 
 ![](../examples/call-hierarchy-ccls.png)
 
-## Metals Doctor
-
-![](../examples/metals-doctor.png)
-
```

Comment:
I'd keep this, like "Flutter debugging" above from `lsp-dart`

Reply:
I thought to add this to lsp-metals readme, but can keep here as well.

Reply:
yes, both are ok :)